### PR TITLE
NL: Use a score different threshold for multi-sv

### PR DESCRIPTION
**Pull request by pradh**
This is to handle cases where multi-sv is behind very slightly from the single-sv score.  Also don't truncate multi-sv scores.

Before:

image

After:

image

### CODE REVIEW

File: server/lib/nl/utils.py [hunk 1]
```diff
@@ -753,6 +753,14 @@ def is_multi_sv(uttr: nl_uttr.Utterance) -> bool:
...
   top_sv_score = uttr.detection.svs_detected.sv_scores[0]
   top_multi_sv_score = uttr.multi_svs.candidates[0].aggregate_score
-  return top_multi_sv_score > top_sv_score
+
+  # Prefer multi-sv when the scores are higher or up to a score differential.
+  if top_multi_sv_score > top_sv_score:
+    return True
+  if (top_sv_score - top_multi_sv_score <=
+      shared_constants.MULTI_SV_SCORE_DIFFERENTIAL):
+    return True
```
Comment (jehangiramjad) [inline server/lib/nl/utils.py:765]: nit: may be combine both these conditions in one statement?

File: nl_server/embeddings.py [hunk 2]
```diff
@@ -38,10 +39,6 @@
 # Scores below this are ignored.
 _SV_SCORE_THRESHOLD = 0.5
 
-# If the difference between successive scores exceeds this threshold, then SVs at
-# the lower score and below are ignored.
-_MULTI_SV_SCORE_DIFFERENTIAL = 0.05
-
 _NUM_CANDIDATES_PER_NSPLIT = 3
```
Comment (jehangiramjad) [inline nl_server/embeddings.py:42]: nit: given that the other constant was moved to the constants file, why not also this?

**Review by jehangiramjad — APPROVED** [review]
Nice!

File: server/lib/nl/utils.py (hunk 1 above)
Comment (pradh) [reply server/lib/nl/utils.py]: Done/

File: nl_server/embeddings.py (hunk 2 above)
Comment (pradh) [reply nl_server/embeddings.py:42]: That is a constants file in shared lib, so only having stuff shared between nl_server and server there.

**Review by pradh — COMMENTED** [review]
Thanks for the review!

**Pull request merged**